src/components/dropdown.tsx

```diff
@@ -134,6 +139,14 @@ const Dropdown = ({
   }
   useOuterClick(dropdownRef);
 
+  const labelValue = (value: string | boolean) => {
+    if (typeof value === 'string') {
+      return value;
+    } else {
+      return value ? 'Yes' : 'No';
+    }
+  };
+
```

Comment:
This is a really poor solution to this issue, but at this point there's no use undoing it.

Reply:
Had to deal with boolean values somehow

Reply:
It would require a dive to update the whole logic of the dropdown

Reply:
Or make a boolean component?

Reply:
It could have been done in `filterInto` by doing the boolean comparison for that field there.

Reply:
```ts
const feedback = feedbackField.value === 'Yes';
```